### DISCUSSION TITLE
Moonraker Timelapse

### DISCUSSION THREAD

**Discussion by Anonoutlaw187** [Guilouz/Creality-Helper-Script-Wiki#536] [Issues]
![image](https://github.com/Guilouz/Creality-Helper-Script-Wiki/assets/156624108/d7025ae4-cb0d-443e-9c01-931965951612)

**Comment by Anonoutlaw187** [comment]
Is there a way for the for the install helper to check the files on the system and reinstall what needs to be installed again.

**Reply by Guilouz** [reply]
That's already the script does.

Your moonraker.conf try to load timelapse but you have not timelapse file in /usr/data/moonraker/moonraker/moonraker/components/timelapse.py
